server/handlers/project_command_output_handler.go

```diff
@@ -51,8 +52,9 @@ func NewProjectCommandOutputHandler(projectCmdOutput chan *models.ProjectCmdOutp
 
 func (p *DefaultProjectCommandOutputHandler) Send(ctx models.ProjectCommandContext, msg string) {
 	p.ProjectCmdOutput <- &models.ProjectCmdOutputLine{
-		ProjectInfo: ctx.PullInfo(),
-		Line:        msg,
+		ProjectInfo:     ctx.PullInfo(),
+		ClearBuffBefore: true,
```

Comment:
You don't want to set this to true everytime you send a message. You want to clear the buffer before starting terraform process on a project. You can create a `Clear(ctx models.ProjectCommandContext, msg string)` function that will set the flag to `true`